app/models/task.rb

```diff
@@ -2,4 +2,17 @@
 
 class Task < ActiveRecord::Base
   validates :name, presence: true
+  validates :completed, on: :create, acceptance: { accept: false, message: 'Task cannot be marked as completed when created.' }
```

Comment:
this is cool, i have never used the `acceptance` validator. 

I checked the rails docs and it says that it ensures the value on the form is checked/or in this case not checked. 

I see you have validates completed on create, so i guess this acceptance validator will only be applied on create? 

Can you think of any other ways to enforce this validation on the DB level?

Reply:
This is also my first time to use <code>acceptance</code> when I accidentally found it in Rails document. It's only applied on create as the user story required. I added default value for tasks' completed column to enforce this validation.

Reply:
nice!